app/controllers/api/base_controller.rb

```diff
@@ -132,4 +132,8 @@ def render_bad_request(error = "bad request")
     error = error.message if error.is_a?(Exception)
     render json: { error: error.to_s }, status: :bad_request
   end
+
+  def owner?
```

Comment:
Any reason to keep it in base controller? What about to move this to ApiKey model (using delegator?) and use it in before_action inline block?

Reply:
it's needed to override the method in ApplicationController, since `current_user` isn't used in the API controllers